.github/workflows/build_and_test.yml

```diff
@@ -87,6 +91,7 @@ jobs:
             sparkr=`./dev/is-changed.py -m sparkr`
             tpcds=`./dev/is-changed.py -m sql`
             docker=`./dev/is-changed.py -m docker-integration-tests`
+            infra_image=`./dev/is-changed.py -m infra-image`
```

Comment:
hmmm .. actually quick question. would that still build the base image when there are other changes together with Dockerfile? Do we return true when `root` module is detected?

Reply:
We could merge this in the morning of some day (such as tmr), I'll keep watching.